assignment5.js

```diff
@@ -0,0 +1,27 @@
+let files = ["./f1","./f2","./f3"], i = 0;
+fs = require('fs');
+function reader(files) {
+  return new Promise(function(resolve, reject) {
+    fs.readFile(files[i], 'utf8' , function (err,data) {
+      if (err) {
+      reject(err);
+    }
+    resolve(data);
+    })
+  });
+}
+
+reader(files).then(function(fromResolve) {
+  console.log(fromResolve);
+  console.log("===========================");
+  i++;
+  return reader(files);
```

Comment:
instead of incrementing `i` every time and passing the whole array of files every time, better pass the name of the file to read as a parameter.